### PR TITLE
HB billing and sub

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

Adds new billing and subscription docs.

- [Billing deploy preview](https://deploy-preview-18829--docsdocker.netlify.app/billing/build-billing/)
- [Subscription deploy preview](https://deploy-preview-18829--docsdocker.netlify.app/subscription/build-details/)

Also added to the deactivate account prereq list:
- [Deploy preview](https://deploy-preview-18829--docsdocker.netlify.app/docker-hub/deactivate-account/)

### Related issues (optional)

Closes JIRA https://docker.atlassian.net/browse/ENGDOCS-1877

### CODE REVIEW

File: content/billing/build-billing.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Docker Cloud Build billing
+description: Learn how to buy a Docker Cloud Build subscription.
+keywords: payments, billing, subscription, build, cloud, cloud build, remote builder
+---
+
+Cloud Build is a next-gen build tool that enables improved build speed, a shared build cache, and native multi-platform builds. Cloud Build is available for free with an existing Docker subscription. To buy additional seats and increase your available build minutes, you can buy a subscription to enhance your builds.
```
Comment (dvdksn) [inline content/billing/build-billing.md:7]:
```suggestion
Cloud Build is a next-gen build tool that enables improved build speed, a shared build cache, and native multi-platform builds. Cloud Build is available for free with an existing Docker subscription. To get additional seats and increase your available build minutes, you can buy a subscription to enhance your builds.
```

File: content/billing/build-billing.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+In this section, learn how to set up and manage billing for your Cloud Build subscription. To learn about the subscription tiers, see [Cloud Build subscriptions and features](../subscription/build-details.md).
```
Comment (dvdksn) [inline content/billing/build-billing.md:9]: suggestion

```suggestion
In this section, learn how to set up and manage billing for your Cloud Build subscription. For more information about subscription tiers, see [Cloud Build subscriptions and features](../subscription/build-details.md).
```

File: content/billing/build-billing.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+You can downgrade your subscription at any time.
+
+1. Sign in to your Docker account and go to the [Docker Cloud Build dashboard](https://www.build.docker.com).
+2. Select **Settings and billing**.
+3. Select **Change plan**.
+4. Select **Downgrade**.
+5. Follow the on-screen instructions then select **Confirm downgrade**.
+
+Your subscription will remain active until your next billing cycle.
```
Comment (dvdksn) [inline content/billing/build-billing.md:41]: suggestion

```suggestion
Your subscription remains active until your next billing cycle.
```

**Review by dvdksn — APPROVED** [review]
LGTM. Left a couple suggestions, not blocking

**Review by dvdksn — DISMISSED** [review]

**Comment by sdidenko22** [comment]
A few comments:
1. Billing Deploy Preview. The links to the Dashboard and the "subscriptions and features" at the top of the billing docs page seem to be broken. When I click on either the first one or the second one it appends www to the URL and doesn't open.

2. Subscription deploy preview. Remove the bullet "Option to buy an additional seat" from Personal and Pro tiers. They are limited to one seat. They can pay $5 a month and get additional 200 minutes but not an extra seat.

3. "Can buy extra buckets of minutes at any time". Minutes are only available to purchase by customers with at least one paid seat.

**Review by dvdksn — APPROVED** [review]

**Pull request merged**